### DISCUSSION TITLE
Does this plugin require a Tex distribution (eg TexLive)?

### DISCUSSION THREAD

**Discussion by AFlowOfCode** [artisticat1/obsidian-latex-suite#324] [Q&A] [answered]
I already have TexLive installed so I wanted to check whether or not it silently connected under the hood & is actually a requirement to run. 

It doesn't seem like it is actually being used, but since I'm about to uninstall the distro I wanted to check first to make sure that doing so won't hobble this excellent plugin. It really combines everything I was looking for in terms of being able to quickly & easily incorporate mathematical symbols in a regular text editor.

Thanks so much!

**Comment by Comprehensive-Jason** [comment, marked as answer]
This plugin does not need Tex. Obsidian uses built-in [MathJax](https://www.mathjax.org/) for math, which technically isn't Latex but is mostly compatible with Latex syntax-wise while having [near-equivalent extensions](https://docs.mathjax.org/en/latest/input/tex/extensions/index.html) .
